apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/OpenOrders.tsx

```diff
@@ -218,37 +214,30 @@ export const OpenOrders = () => {
       tokenPrice,
       tokenAmount,
     } = order
+    setIsLoading(true)
     const totalReturned = `${fromTokenAmount} ${fromTokenInfo?.ticker}`
     const collateralUtxo = await getCollateralUtxo()
 
-    setBottomSheetState({
-      height: 400,
-      openId: id,
-      title: strings.listOrdersSheetTitle,
-      content: (
-        <Suspense fallback={<ModalLoadingState />}>
```

Comment:
Seems like there are issues when mixing suspense + reducer... I will take a look when the modal migration starts